ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```diff
@@ -148,6 +148,13 @@ retry_with_timeout() {
   done
 }
 
+# print RTC logs
+print_time() {
+  oc debug node/cnfdf30.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
+  oc debug node/cnfdf31.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
+  oc debug node/cnfdf32.telco5gran.eng.rdu2.redhat.com -- chroot /host sh -c "date;sudo hwclock"
```

Comment:
Can we probably discover nodes and loop over them? We'll need to fix every time if nodes names are changed, or different infrastructure will be set up.

Reply:
fixed